packages/editor-ui/src/App.vue

```diff
@@ -15,15 +15,26 @@
 
 <script lang="ts">
 import Vue from 'vue';
+import { mapGetters } from 'vuex';
 import Telemetry from './components/Telemetry.vue';
+import { HIRING_BANNER } from './constants';
 
 export default Vue.extend({
 	name: 'App',
 	components: {
 		Telemetry,
 	},
+	computed: {
+		...mapGetters({
+			isInternalUser: 'settings/isInternalUser',
+		}),
```

Comment:
nitpick - can also be rewritten as
```suggestion
		...mapGetters('settings', ['isInternalUser']),
```